api/client/ps/custom.go

```diff
@@ -65,7 +65,9 @@ func (c *containerContext) Image() string {
 		return "<no image>"
 	}
 	if c.trunc {
-		return stringutils.Truncate(c.c.Image, 12)
+		if len(c.c.Image) == 64 {
```

Comment:
I think the possibility that the name is just 64-characters is so low. I think no one
will named a image with such a long name. :-)